media-fonts/ocr/additional-files/fontconfig-relaod-cache.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+fc-cache -f
```

Comment:
This should probably check whether fc-cache is available before calling it because our font packages don't depend on fontconfig (it is optional on Haiku and not used by native apps IIRC).

So, probably something like this:

```sh
type fc-cache >&/dev/null && fc-cache -f
```

Reply:
If fc-cache isn't available it wouldn't do anything I guess like that?
Maybe a fontconfig requirement in REQUIRES to make sure it's installed?

Reply:
Updated per suggestion :)

Reply:
If fontconfig is not installed, the post-install script would fail. I don't know how the package kit would handle that case ...

Reply:
If one would put it in REQUIRES it would be downloaded and installed before the build starts, so that way you can ensure that it is installed:
```
REQUIRES="
	cmd:fc_cache
	"
```

Reply:
that would be an alternative, but I don't know if that is wanted in general. app_server doesn't need or use fontconfig, is that correct?

Reply:
No idea there, but trying to "uninstall" it leads to several packages wanting to uninstall (among haiku_devel package)

Reply:
As I said, this is probably not relevant at all for native apps (like Pe) but only for ported apps that actually use fontconfig (like xetex).